src/NHibernate.Test/Hql/SimpleFunctionsTest.cs

```diff
@@ -218,15 +178,7 @@ public void ClassicAvg()
 			args.Clear();
 			args.Add("va1");
 			args.Add("va2");
-			try
-			{
-				SqlString t = caf.Render(args, factoryImpl);
-				Assert.Fail("No exception 2 argument without <setquantifier>:" + t);
```

Comment:
But why assert messages were removed? It was merged too fast

Reply:
I'm happy to create another PR. Do you think these messages added valuable context?

Reply:
Imo, not really

Reply:
Now it's not clear at all why ClassicAvg test should fail with `QueryException`

Reply:
Most test names here don't provide any context. So not sure why you think it's not valuable.

Reply:
@bahusoid I created another PR here: https://github.com/nhibernate/nhibernate-core/pull/2124
We can continue the conversation there.